src/Core/src/Layouts/GridLayoutManager.cs

```diff
@@ -316,6 +318,76 @@ void ResolveSpan(Definition[] definitions, int start, int length, double spacing
 
 				return top;
 			}
+
+			void ResolveStars(Definition[] defs, double availableSpace, Func<Cell, bool> cellCheck, Func<Size, double> dimension)
+			{
+				// Count up the total weight of star columns (e.g., "*, 3*, *" == 5)
+
+				var starCount = 0;
+
+				foreach (var definition in defs)
+				{
+					if (definition.IsStar)
+					{
+						starCount += (int)definition.GridLength.Value;
+					}
+				}
+
+				if (starCount == 0)
+				{
+					return;
+				}
+
+				double starSize = 0;
+
+				if (double.IsInfinity(availableSpace))
+				{
+					// If the available space we're measuring is infinite, then the 'star' doesn't really mean anything
+					// (each one would be infinite). So instead we'll use the size of the actual view in the star row/column.
+					// This means that an empty star row/column goes to zero if the available space is infinite. 
+
+					foreach (var cell in _cells)
+					{
+						if (cellCheck(cell)) // Check whether this cell should count toward the type of star value were measuring
+						{
+							// Update the star width if the view in this cell is bigger
+							starSize = Math.Max(starSize, dimension(_grid.Children[cell.ViewIndex].DesiredSize));
+						}
+					}
+				}
+				else
+				{
+					// If we have a finite space, we can divvy it up among the full star weight
+					starSize = availableSpace / starCount;
+				}
+
+				foreach (var definition in defs)
+				{
+					if (definition.IsStar)
+					{
+						// Give the star row/column the appropriate portion of the space based on its weight
+						definition.Size = starSize * (int)definition.GridLength.Value;
```

Comment:
What happens if I have a column set of `3*, 100, 1*`? I can't seem to spot the case where you are also excluding the explicit sizes.

Reply:
I just added a unit test to demo that situation and verify that it's accounted for - it's at the end of GridLayoutManagerTests.

GridLayoutManager has its own nested classes for tracking Row/Column info; in their constructors (check out line 500 or so), they check for absolute size values and set them immediately. That's where the absolute sizes get accounted for. 

In `MeasureCells()`, the loop just ignores all cells that are absolute in both dimensions; they're already accounted for. That's also where Auto stuff that doesn't span more than one row/column gets handled. The stuff that spans is tracked to be resolved later. (Stuff that spans multiple rows/columns and includes a Star _isn't_ accounted for yet - that's among the use cases that aren't covered yet.)

So in ResolveStarColumns and ResolveStarRows, all the other stuff has already been dealt with; that's why you're not seeing it referenced there. Again, some of this _will_ change when we update it to account for the mixed spans.